examples/webaudio.js

```
@@ -2,72 +2,67 @@
+  // Connect the filters to the audio output
+  equalizer.connect(audioContext.destination)
 
-    // Connect the filters to the audio output
-    equalizer.connect(audioContext.destination)
-  },
-  { once: true },
-)
+  sliders.forEach((slider, i) => {
+    const filter = filters[i]
+    filter.gain.value = slider.value
+    slider.oninput = (e) => (filter.gain.value = e.target.value)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)